mm2src/lp_ordermatch/simple_market_maker.rs

```diff
@@ -259,6 +259,28 @@ async fn create_single_order(cfg: SimpleCoinMarketMakerCfg, key_trade_pair: Stri
     if cfg.check_last_bidirectional_trade_thresh_hold.unwrap_or(false) {
         vwap_apply(&mut calculated_price).await;
     }
+    let req = SetPriceReq {
+        base: cfg.base.clone(),
+        rel: cfg.rel.clone(),
+        price: calculated_price,
+        max: cfg.max.unwrap_or(false),
+        volume: Default::default(),
+        min_volume: None,
+        cancel_previous: true,
+        base_confs: cfg.base_confs,
+        base_nota: cfg.base_nota,
+        rel_confs: cfg.rel_confs,
+        rel_nota: cfg.rel_nota,
+        save_in_history: true,
+    };
```

Comment:
I need to check if it's possible to do a function `new` that just take a bot `cfg` and the `calculated_price`